src/Common/src/System/Security/Cryptography/DerSequenceReader.cs

```diff
@@ -194,6 +194,30 @@ internal string ReadOidAsString()
             return builder.ToString();
         }
 
+        internal byte[] ReadBitString()
+        {
+            // Bitstrings are primitives with tag and length, followed by a number
+            // of unused bits in the last byte, followed by the 
+            EatTag(DerTag.BitString);
+            int length = EatLength();
+
+            if (length == 0)
+                return Array.Empty<byte>();
+
+            // We skip over the number of unused bits as it is not needed, and allocate one less 
+            // byte as this is not included in the bitstring. According to DER, all trailing padding must be
+            // composed of zeros, so there's no need to clear these bits. In case this encoder is expanded to BER
+            // or other encodings it would be necessary to read this number and clear the corresponding bits.
+            _position += 1;
+
+            byte[] octets = new byte[length-1];
+            Buffer.BlockCopy(_data, _position, octets, 0, length-1);
```

Comment:
Since you do `length - 1` twice, I'd either save it to a local variable, or here just use octets.Length.